mail_composer_cc_bcc/tests/test_mail_cc_bcc.py

```diff
@@ -31,6 +35,7 @@ def open_mail_composer_form(self):
             "default_partner_ids": test_record.ids,
             "default_model": test_record._name,
             "default_res_id": test_record.id,
+            "mail_notify_force_send": True,
```

Comment:
can you add a comment, something like # get consistent tests results even if `mail_post_defer` is installed